src/ununpack/agent/utils.c

```diff
@@ -21,6 +21,12 @@
 #include "ununpack.h"
 #include "externs.h"
 #include "regex.h"
+#include "sha2.h"
+
+#define NEW_SHA256
+#ifdef NEW_SHA256
```

Comment:
Do we still need these macros?

Reply:
No we don't,
I'm sorry, that's a leftover, and should be removed

Reply:
I remove it in latest commit